homeassistant/components/vallox/sensor.py

```diff
@@ -140,20 +150,40 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
         icon="mdi:fan",
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=PERCENTAGE,
-        sensor_type=ValloxFanSpeedSensor,
+        entity_type=ValloxFanSpeedSensor,
+    ),
+    ValloxSensorEntityDescription(
+        key="extract_fan_speed",
+        name="Extract fan speed",
+        metric_key="A_CYC_EXTR_FAN_SPEED",
+        icon="mdi:fan",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=FREQUENCY_HERTZ,
+        entity_type=ValloxFanSpeedSensor,
+        entity_registry_enabled_default=False,
+    ),
+    ValloxSensorEntityDescription(
```

Comment:
Should this have an entity category?

Reply:
We have "diagnostic" everywhere in ValloxSensorEntity class.